components/treelist/accessibility/wai-aria-support.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor TreeList Documentation | TreeList  Accessibility
+description: "Get started with the Telerik UI for Blazor TreeList and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.1."
+tags: telerik,blazor,accessibility,wai-aria,wcag
+slug: treelist-wai-aria-support 
+position: 50 
+---
+
+# Blazor TreeList Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
+Out of the box, the Telerik UI for Blazor TreeList provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The TreeList is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2  AA](https://www.w3.org/TR/WCAG22/) standards](https://www.w3.org/TR/WCAG22/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
+
+## WAI-ARIA
+
+
+This section lists the selectors, attributes, and behavior patterns supported by the component and its composite elements, if any.
+
+
+The TreeList is a composite component that consists of 2 logically separated structural elements:
+
+
+ - Toolbar (`role=toolbar`);
+ - Tree Grid (`role=treegrid`);
+
+### TreeList Toolbar
+
+
+TreeList Toolbar follows the specification of the ToolBar component.
+
+[ToolBar accessibility specification]({{toolbar_a11y_link}})
+
+| Selector | Attribute | Usage |
+| -------- | --------- | ----- |
+| `.k-grid-toolbar` | `role=toolbar` | The toolbar is a collection of command buttons and inputs. |
+|  | `aria-label` | Clarifies the purpose of the toolbar. |
+|  | `aria-controls=.k-grid-aria-root id` | Pointing to the id of the element with `role=treegrid`. |
+
+### Tree Grid
+
+
+The element with `role=treegrid` must not include the ToolBar element as it does not belong to the `role=treegrid` element itself. It implements the ARIA specification for the Grid component with some exceptions listed below.
+
+[Grid accessibility specification]({{grid_a11y_link}})
+
+
+The below table lists the ARIA requirements for the TreeList which are not present in the Grid specification:
+
+| Selector | Attribute | Usage |
+| -------- | --------- | ----- |
+| `.k-treelist .k-grid-aria-root` | `role=treegrid` | Sets the proper role for the TreeList component. |
+| `.k-table-tbody .k-table-row` | `aria-expanded=true/false` | Set on the currently expanded row(s). |
```

Comment:
Shouldn't this be `Sets` for consistency? However, since this is automatically generated, it might be somewhat difficult to fix. If so, ignore this suggestion.
```suggestion
| `.k-table-tbody .k-table-row` | `aria-expanded=true/false` | Sets on the currently expanded row(s). |
```